res/css/views/auth/_AuthPage.pcss

```diff
@@ -29,4 +29,13 @@ Please see LICENSE files in the repository root for full details.
     @media only screen and (max-width: 480px) {
         margin-top: 0;
     }
+
+    /* Extra class to indicate that the component is providing its own blurring,
+     * so we don't apply ours */
```

Comment:
... nor a box-shadow, apparently.

What exactly does "blurring" mean, in this context?

This whole mechanism of overriding some of the inline styles by addition of another class feels a bit opaque. (Blurry, if you will.) It feels like maybe we should instead have a class which *enables* the blurring, and which `AuthPage` enables by default, but which you can disable via a boolean property on `AuthPage` (instead of the `modalClasses` prop). WDYT?

Reply:
> This whole mechanism of overriding some of the inline styles by addition of another class feels a bit opaque. (Blurry, if you will.) It feels like maybe we should instead have a class which _enables_ the blurring, and which `AuthPage` enables by default, but which you can disable via a boolean property on `AuthPage` (instead of the `modalClasses` prop). WDYT?

Done.  I had originally used the class property to make it more flexible, but the boolean is clearer what the intent is.